docs/data-feeds/proof-of-reserve/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: "Proof of Reserve Feeds"
+permalink: "docs/data-feeds/proof-of-reserve/"
+whatsnext: {
+  "Learn how to read answers from data feeds":"/docs/get-the-latest-price/",
+  "Find contract addresses for PoR feeds":"/docs/data-feeds/proof-of-reserve/addresses/"
+}
+---
+
+Proof of reserve feeds provide the status of the reserves that back several asset tokens on-chain. These feeds operate the same way as other Chainlink Data Feeds. To learn how to use these feeds, see the [Using Data Feeds](/docs/get-the-latest-price/) guide.
+
+To find a list of available proof of reserve feeds, see [Proof of Reserve Feed Addresses](/docs/data-feeds/proof-of-reserve/addresses/).
+
+**Topics**
+
+- [Types of Proof of Reserve feeds](#types-of-proof-of-reserve-feeds)
+
+## Types of Proof of Reserve feeds
+
+Reserves are available for both cross-chain assets and off-chain assets. Token issuers prove the reserves for their assets through several different methods. 
+
+- [Cross-chain reserves](#cross-chain-reserves):
+  - Wallet address manager: The project uses the [IPoRAddressList]() wallet address manager contract and self-attests to which addresses they own.
```

Comment:
link to IPoRAddressList needs to be fixed

Reply:
TBD

Reply:
Added a link to the contract.